### DISCUSSION TITLE
How to reduce `.terragrunt-cache` size

### DISCUSSION THREAD

**Discussion by marinalimeira** [gruntwork-io/knowledge-base#243] [Help] [answered]
Our base infrastructure repo with only 5 accounts/30 accounts is already 15GB in size, locally, due to .terragrunt-cache folders in every module. Is there any smart way to reduce the total size of the cache folders?

**Comment by yorinasub17** [comment, marked as answer]
This is unfortunately a known issue that we haven't had a chance to tackle. The issue ticket in `terragrunt` for this feature request is https://github.com/gruntwork-io/terragrunt/issues/561. I recommend taking a look at some of the comments on the issue, as some of our open source users have found ways to workaround this by using a centralized download directory (specifically, [this comment](https://github.com/gruntwork-io/terragrunt/issues/561#issuecomment-1032170292)).

**Reply by marinalimeira** [reply]
Thanks Yori!

**Reply by LuigiClemente-Awin** [reply]
Thanks both! :-)
